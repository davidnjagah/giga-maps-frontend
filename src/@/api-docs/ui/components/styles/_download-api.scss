// .download-api-signUp-modal {
//   display: flex;

//   >.cds--modal-container {
//     background: #FFFFFF;
//     width: 46rem;
//     height: auto;
//   }
// }

// .download-api-signUp-modal-body {
//   width: 100%;
//   margin-bottom: 0rem;
//   padding-right: 0;
//   display: flex;
//   flex-direction: row;

//   >.download-api-signUp-modal-body-left {
//     width: 55%;
//     margin-right: 1rem;
//   }

//   >.download-api-signUp-modal-body-right {
//     width: 45%;

//     >svg {
//       height: 21%;
//       margin-left: 2rem;
//     }
//   }

// }

// .download-api-signUp-form-field {
//   margin-top: 1.5rem;
// }

// .download-api-signUp-button {
//   margin-top: 1.5rem;
//   width: 100%;
//   max-width: 100%;
// }

// .download-api-signUpWith-sso {
//   margin-top: 1.5rem;

//   >p {
//     color: var(--primary-black-80-giga-dark-grey, #474747);
//     font-family: Open Sans;
//     font-size: 0.875rem;
//     font-style: normal;
//     font-weight: 400;
//     line-height: 1.125rem;
//   }
// }

// .download-api-signUpWith-sso-button {
//   margin-top: 1rem;
//   width: 100%;
//   max-width: 80%;
// }

// .download-api-signUp-modal-info {
//   background: linear-gradient(180deg, rgba(244, 244, 244, 0) -15.58%, rgba(47, 125, 252, 0.08) 100%);
//   padding: 0rem 0rem 0rem 2rem;
//   height: 75%;

//   >h5 {
//     color: var(--text-text-tertiary, #6F6F6F);
//     font-family: Open Sans;
//     font-size: 0.875rem;
//     font-style: normal;
//     font-weight: 400;
//     line-height: 1.125rem;
//     margin-top: 1rem;
//   }

//   >h3 {
//     color: var(--text-text-secondary, #474747);
//     font-family: Open Sans;
//     font-size: 1.25rem;
//     font-style: normal;
//     font-weight: 500;
//     line-height: 1.75rem;
//     margin-top: 0.5rem;
//   }

//   >svg {
//     margin-top: 0.25rem;
//   }

//   >h4 {
//     color: var(--text-text-tertiary, #6F6F6F);
//     font-family: Open Sans;
//     font-size: 1rem;
//     font-style: normal;
//     font-weight: 500;
//     line-height: 1.5rem;
//     margin-top: 1.5rem;
//   }

//   >p {
//     color: var(--primary-black-80-giga-dark-grey, #474747);
//     font-family: Open Sans;
//     font-size: 0.875rem;
//     font-style: normal;
//     font-weight: 400;
//     line-height: 1.125rem;
//     margin-top: 10rem;

//     >a {
//       margin-left: 0.25rem;
//     }
//   }
// }