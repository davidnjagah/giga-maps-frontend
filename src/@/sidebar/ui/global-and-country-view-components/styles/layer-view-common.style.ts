import { styled } from "styled-components";

import { Scroll } from '@/scroll';

export const LayerHeader = styled.div` 
  display: flex;
  justify-content: space-between;
  align-items: center;
  
  svg{
    fill:${props => props.theme.text} !important;
  }
  
`

export const SchoolInfoSection = styled.div`
  margin-top: 1.25rem;
  margin-left: 1rem;

.highlight-number {
  color: var(--main-color-green);
  font-size: 2rem;
  font-weight: 500;
}

.descriptons {
  color: var(--primary-color-primary);
  font-size: 0.75rem;
}
`

export const SchoolConnectivityScroll = styled(Scroll)`
  max-height: calc(100vh - 19rem);

  @media (max-width:575px){
    max-height: calc(100vh - 31rem);
  }
`

export const CoverageLayerScroll = styled(Scroll)`
max-height: calc(100vh - 19rem);

  @media (max-width:575px){
    max-height: calc(100vh - 31rem);
  }
  `
export const FilterIconAndName = styled.div`
display:flex;
align-items:center;
justify-content: space-between;
padding: 0.55rem 1rem;
.cds--tooltip-content{
  font-size: 0.8rem;
  max-width: 10rem;
}
.layer-heading {
    margin: 0;
    color: ${props => props.theme.text};
    font-size: 0.875rem;
    font-style: normal;
    font-weight: 500;
    line-height: 1.5rem;
    font-family: 'Open Sans';
    margin-left: .5rem;
  }

  svg{
    fill:${props => props.theme.text};
  }

 .info-icon{
   button{
  border: none;
    background: inherit;
    margin-top: 0.3rem;
    margin-left: 0.5rem;
    svg{
      fill:#7E7E7E;
      width: 0.75rem;
      height: 0.75rem;
    }
}
}
`

export const SidebarFooterGigalayerContainer = styled.div`
position: fixed;
bottom: 1.8rem;
display: flex;
align-items: flex-start;
justify-content: space-between;
width: inherit;
z-index: 1;
border-top: 1px solid ${props => props.theme.schoolListBack};
background:${props => props.theme.main};

.cds--btn--ghost:not([disabled]) svg {
    fill: var(--cds-icon-primary, #161616);
    color: #9e9e9e;
    height: 1rem;
    margin-right: 0.5rem;
    width: 1rem;
}

#connectivityID {
  margin: 0;
    padding-left: 0.2rem;
    padding-right: 0.2rem;
}

.sidebar-worldview-gigaIcon{
  padding: 0;
    display: flex;
    align-items: center;
}

.sidebar-footer-gigalayer-icons-container {
  display: flex;
  flex-direction: row;
  align-items: center;
  // width: 100%;
  padding: 0.5rem;
  padding-right:0rem;

  .cds--popover-caret{
    background: ${props => props.theme.main};
  }
}

@media (max-width:768px){
  bottom: 0rem;
  width:100%;
}
`