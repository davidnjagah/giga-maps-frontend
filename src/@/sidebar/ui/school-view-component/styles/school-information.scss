// .sidepanel-schoolview-schoolInfo {
//   display: grid;
//   grid-template-columns: 1fr 1fr;
//   margin: 0 auto;
//   // padding: 0rem 0rem 0.88rem 0rem;
//   width: inherit;
//   // max-height: 11.2rem;
//   overflow-y: auto;
//   width: 100%;
// }

// .sidepanel-schoolview-schoolInfo-item {
//   padding: 0.75rem 0.5rem;
//   overflow: hidden;
//   word-wrap: break-word;
//   padding-left: 0;
// }

// .sidepanel-schoolview-schoolInfo-item p {
//   color: var(--text-text-tertiary, #6f6f6f);
//   font-family: Open Sans;
//   font-size: 0.75rem;
//   font-style: normal;
//   font-weight: 400;
//   line-height: 1rem;
//   margin-bottom: 0.25rem;
//   /* 133.333% */
// }

// .sidepanel-schoolview-schoolInfo-item span {
//   color: var(--text-text-primary, #222);
//   font-family: Open Sans;
//   font-size: 0.875rem;
//   font-style: normal;
//   font-weight: 400;
//   line-height: 1.125rem;
//   /* 128.571% */
// }

// .sidepanel-schoolview-schoolInfo-for-scroll {
//   // flex: 1;
//   // overflow-y: auto;
//   // width: inherit;
//   max-height: calc(100% - 16rem);
//   // overflow-x: hidden;

//   .side-info-panel-schoolconnectivity-only-selection-layer-container {
//     margin-top: 0.5rem;
//     margin-left: 1rem;
//     margin-bottom: 1.15rem;
//     margin-right: 1rem;
//   }
// }

// .school-connectivity-status-layer.schoolview-single-accordion {
//   position: inherit;
//   margin-top: 0rem;
//   padding: 1rem;
//   background-color: #FAFAFA;

//   >ul {
//     background: var(--layer-layer-01, #FAFAFA) !important;
//     margin-left: 0px;
//     padding-left: 0px;

//     .cds--accordion__item--active.sidepanel-schoolview-schoolInfo-accordionItem,
//     .cds--accordion__item.sidepanel-schoolview-schoolInfo-accordionItem {
//       border-top: none;
//       border-bottom: none;
//     }

//     .cds--accordion__title {
//       // margin: 0.88rem 0rem 0rem 1rem;
//       padding-right: 0;
//     }

//   }
// }

// .school-connectivity-status-layer.multischoolview-single-accordion {
//   position: relative;
//   bottom: 3rem;

//   >.cds--accordion.cds--accordion--end {
//     background: #fff;

//     >.cds--accordion__item.cds--accordion__item--active {
//       >.cds--accordion__content {
//         padding: 0;
//         margin-left: 0;

//         .sidebar-connectivity-status-heading-text {
//           display: flex;
//           justify-content: space-between;
//           align-items: center;
//           height: 1rem;
//           width: 95%;
//           margin-left: 1rem;
//           margin-top: 0.88rem;
//           // margin-bottom: 0.88rem;
//         }

//         .side-info-panel-schoolconnectivity-only-selection-layer-container {
//           margin-left: 1rem;
//         }

//       }
//     }

//     >.cds--accordion__item {
//       margin-top: 0;
//       border: none;

//       >.cds--accordion__heading:hover::before {
//         background-color: #fff
//       }


//       >.cds--accordion__heading {
//         // width: 21rem;

//         svg {
//           position: relative;
//           top: -1rem;
//         }

//         .cds--accordion__title {
//           padding-bottom: 0.5rem;
//           width: 23rem;
//           white-space: nowrap;
//           margin: 0;
//           direction: ltr;
//           padding-left: 0.125rem;

//           .side-info-panel-multischoolview-single-accordion-title {
//             position: relative;
//             left: 2.5rem;
//             width: 17rem;
//             overflow: hidden;
//             text-overflow: ellipsis;
//             display: block;
//             top: -0.35rem;
//           }

//           .side-info-panel-multischoolview-single-accordion-bottom-info {
//             span {
//               position: relative;
//               left: 0rem;
//             }

//             svg {
//               position: relative;
//               top: 0rem;
//             }
//           }
//         }

//       }

//       >.cds--accordion__heading::before {
//         border: none;
//       }
//     }

//     >.cds--accordion__item.cds--accordion__item--active {
//       margin-top: 0;

//       >.cds--accordion__heading> {
//         svg {
//           position: relative;
//           top: -1rem;
//         }

//         .cds--accordion__title {
//           padding-bottom: 0.5rem;
//           width: 23rem;
//           white-space: nowrap;

//           .side-info-panel-multischoolview-single-accordion-title {
//             position: relative;
//             left: 2.5rem;
//             width: 17rem;
//             overflow: hidden;
//             text-overflow: ellipsis;
//             display: block;
//             top: -7px
//           }

//           .side-info-panel-multischoolview-single-accordion-bottom-info {
//             span {
//               position: relative;
//               left: 0rem;
//             }
//           }
//         }
//       }

//       >.cds--accordion__heading::before {
//         border: none;
//       }
//     }
//   }

//   .sidebar-worldview-layer-connectivityIndicator-container {
//     margin-top: 0.8rem;
//     margin-left: 0;
//     width: 99%;

//     .sidebar-worldview-filterIcon-container {
//       margin-right: 0;
//     }
//   }

//   .side-info-panel-multischoolview-single-accordion-bottom-info {
//     display: flex;
//     padding: 0.5rem 0 0 0;
//     position: relative;
//     left: 0.8rem;
//   }

//   .side-info-panel-multischoolview-single-accordion-bottom-info-first {
//     display: flex;
//     align-items: center;
//     width: 10rem;
//   }

//   .side-info-panel-multischoolview-single-accordion-bottom-info-second {
//     display: flex;
//     align-items: center;
//     margin-left: 0.1rem;
//   }

// }

// .side-info-panel-connectivityIndicator-layer-multischool-selection-container {
//   width: 100%;
//   margin: 0;
//   // padding: 0 1rem;
//   position: relative;

//   .side-info-panel-connectivityIndicator-layer-multischool-selection-checkbox-and-accordion-container {

//     >.cds--form-item.cds--checkbox-wrapper {
//       position: relative;
//       top: 0.8rem;
//       max-width: 10%;
//       margin-left: 0.5rem;

//       >.cds--checkbox-label {
//         width: 22rem;

//         >.cds--checkbox-label-text {
//           position: relative;
//           top: 2rem;
//           left: -2.5em;
//           width: 22rem;
//         }
//       }
//     }

//     .school-connectivity-status-layer.multischoolview-single-accordion {
//       // width: 21rem;
//       position: unset;

//       .sidepanel-schoolview-schoolInfo-for-scroll {
//         .side-info-panel-connectivityIndicator-layer-container {
//           margin: 0;
//         }

//         .school-connectivity-status-layer.schoolview-single-accordion {
//           position: inherit;
//           margin-top: 0.88rem;

//           >ul {
//             background: var(--layer-layer-01, #FAFAFA) !important;
//             margin-left: 0px;
//             padding-left: 0px;

//             .cds--accordion__item--active.sidepanel-schoolview-schoolInfo-accordionItem,
//             .cds--accordion__item.sidepanel-schoolview-schoolInfo-accordionItem {
//               margin-top: 1.53rem !important;
//               border-top: none;
//               border-bottom: none;

//               .cds--accordion__title {
//                 margin-left: 0;
//               }

//               .cds--accordion__content {
//                 position: relative;
//                 left: -0.9rem;
//               }
//             }

//           }
//         }

//         .side-info-panel-schoolconnectivity-only-selection-layer-container {
//           margin-left: 1rem;
//         }
//       }
//     }
//   }

//   >.side-info-panel-multischoolview-single-accordion-bottom-line {
//     width: 100%;
//     height: 1px;
//     background: #dadada;
//     position: relative;
//     // left: -10px;
//   }
// }

// .side-info-panel-connectivityIndicator-layer-multischool-selection-checkbox-and-accordion-container {
//   display: flex;
//   justify-content: flex-start;
// }


// .side-info-panel-connectivityIndicator-layer-multischool {
//   // height: 72vh;
//   max-height: calc(100% - 16rem);

// }

// @media (max-width: 376px) and (min-width: 281px) {
//   .side-info-panel-connectivityIndicator-layer-multischool-selection-container {
//     width: 100%;
//     margin-left: 0.5rem;

//     .side-info-panel-connectivityIndicator-layer-multischool-selection-checkbox-and-accordion-container {
//       width: inherit;

//       >.cds--form-item.cds--checkbox-wrapper {
//         position: relative;
//         top: 0.8rem;
//         max-width: 10%;
//         margin-left: 0.5rem;

//         >.cds--checkbox-label {
//           width: inherit;

//           >.cds--checkbox-label-text {
//             position: relative;
//             top: 2rem;
//             left: -2.5em;
//             width: 100vw;

//             >.school-connectivity-status-layer.multischoolview-single-accordion {
//               width: inherit;

//               >.cds--accordion.cds--accordion--end {
//                 >.cds--accordion__item {

//                   .cds--accordion__heading {
//                     width: inherit;

//                     >.cds--accordion__title {
//                       padding-bottom: 0.5rem;
//                       width: 16rem;


//                       >.side-info-panel-multischoolview-single-accordion-title {
//                         display: block;
//                         width: 14rem;
//                         overflow: hidden;
//                         white-space: nowrap;
//                         text-overflow: ellipsis;
//                       }

//                     }
//                   }
//                 }
//               }
//             }
//           }
//         }
//       }


//     }

//     >.side-info-panel-multischoolview-single-accordion-bottom-line {
//       width: inherit;
//       height: 1px;
//       background: #dadada;
//       position: relative;
//     }
//   }

// }

// @media (max-width: 394px) and (min-width: 376px) {
//   .side-info-panel-connectivityIndicator-layer-multischool-selection-container {
//     width: 100%;
//     margin-left: 0.5rem;

//     .side-info-panel-connectivityIndicator-layer-multischool-selection-checkbox-and-accordion-container {
//       width: inherit;

//       >.cds--form-item.cds--checkbox-wrapper {
//         position: relative;
//         top: 0.8rem;
//         max-width: 10%;
//         margin-left: 0.5rem;

//         >.cds--checkbox-label {
//           width: inherit;

//           >.cds--checkbox-label-text {
//             position: relative;
//             top: 2rem;
//             left: -2.5em;
//             width: 100vw;

//             >.school-connectivity-status-layer.multischoolview-single-accordion {
//               width: inherit;

//               >.cds--accordion.cds--accordion--end {
//                 >.cds--accordion__item {

//                   .cds--accordion__heading {
//                     width: inherit;

//                     >.cds--accordion__title {
//                       padding-bottom: 0.5rem;
//                       width: 21rem;


//                       >.side-info-panel-multischoolview-single-accordion-title {
//                         display: block;
//                         width: 19rem;
//                         overflow: hidden;
//                         white-space: nowrap;
//                         text-overflow: ellipsis;
//                       }

//                     }
//                   }
//                 }
//               }
//             }
//           }
//         }
//       }


//     }

//     >.side-info-panel-multischoolview-single-accordion-bottom-line {
//       width: inherit;
//       height: 1px;
//       background: #dadada;
//       position: relative;
//     }
//   }

// }

// @media (max-width: 415px) and (min-width: 394px) {
//   .side-info-panel-connectivityIndicator-layer-multischool-selection-container {
//     width: 100%;
//     margin-left: 0.5rem;

//     .side-info-panel-connectivityIndicator-layer-multischool-selection-checkbox-and-accordion-container {
//       width: inherit;

//       >.cds--form-item.cds--checkbox-wrapper {
//         position: relative;
//         top: 0.8rem;
//         max-width: 10%;
//         margin-left: 0.5rem;

//         >.cds--checkbox-label {
//           width: inherit;

//           >.cds--checkbox-label-text {
//             position: relative;
//             top: 2rem;
//             left: -2.5em;
//             width: 100vw;

//             >.school-connectivity-status-layer.multischoolview-single-accordion {
//               width: inherit;

//               >.cds--accordion.cds--accordion--end {
//                 >.cds--accordion__item {

//                   .cds--accordion__heading {
//                     width: inherit;

//                     >.cds--accordion__title {
//                       padding-bottom: 0.5rem;
//                       width: 22rem;


//                       >.side-info-panel-multischoolview-single-accordion-title {
//                         display: block;
//                         width: 20rem;
//                         overflow: hidden;
//                         white-space: nowrap;
//                         text-overflow: ellipsis;
//                       }

//                     }
//                   }
//                 }
//               }
//             }
//           }
//         }
//       }


//     }

//     >.side-info-panel-multischoolview-single-accordion-bottom-line {
//       width: inherit;
//       height: 1px;
//       background: #dadada;
//       position: relative;
//     }
//   }

// }

// @media (max-width: 541px) and (min-width: 415px) {
//   .side-info-panel-connectivityIndicator-layer-multischool-selection-container {
//     width: 100%;
//     margin-left: 0.5rem;

//     .side-info-panel-connectivityIndicator-layer-multischool-selection-checkbox-and-accordion-container {
//       width: inherit;

//       >.cds--form-item.cds--checkbox-wrapper {
//         position: relative;
//         top: 0.8rem;
//         max-width: 10%;
//         margin-left: 0.5rem;

//         >.cds--checkbox-label {
//           width: inherit;

//           >.cds--checkbox-label-text {
//             position: relative;
//             top: 2rem;
//             left: -2.5em;
//             width: 100vw;

//             >.school-connectivity-status-layer.multischoolview-single-accordion {
//               width: inherit;

//               >.cds--accordion.cds--accordion--end {
//                 >.cds--accordion__item {

//                   .cds--accordion__heading {
//                     width: inherit;

//                     >.cds--accordion__title {
//                       padding-bottom: 0.5rem;
//                       width: 28rem;


//                       >.side-info-panel-multischoolview-single-accordion-title {
//                         display: block;
//                         width: 26rem;
//                         overflow: hidden;
//                         white-space: nowrap;
//                         text-overflow: ellipsis;
//                       }

//                     }
//                   }
//                 }
//               }
//             }
//           }
//         }
//       }


//     }

//     >.side-info-panel-multischoolview-single-accordion-bottom-line {
//       width: inherit;
//       height: 1px;
//       background: #dadada;
//       position: relative;
//     }
//   }

// }

// .side-info-panel-connectivityIndicator-layer-multischool-selection-checkbox-container {
//   position: absolute;
//   top: 0.6rem;
//   left: 0.8rem;
//   z-index: 999;
//   margin-right: 0.5rem;
// }

// .side-info-panel-school-container {
//   margin: 0rem 1rem 0rem 1rem;
//   display: flex;
//   justify-content: space-between;
//   align-items: center;
//   margin-bottom: 1.53rem;
// }

// .sidepanel-schoolview-schoolInfo-title {
//   width: 100%;

//   >h1 {
//     color: var(--primary-black-100-giga-black, #222);
//     font-family: Open Sans;
//     font-size: 0.875rem;
//     font-style: normal;
//     font-weight: 500;
//     line-height: 1.25rem;
//     letter-spacing: 0.01rem;
//   }
// }