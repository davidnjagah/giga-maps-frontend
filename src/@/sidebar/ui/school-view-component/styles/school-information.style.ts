import styled from "styled-components";

export const StatisticsStatus = styled.span<{ $color: string; }>`
  color: ${props => props.$color ? props.$color : '#46C66D'} ;
  text-transform: capitalize;
font-size: 0.75rem;
font-weight: 400;
line-height: 2.125rem;
`

export const StatisticsStatusLg = styled(StatisticsStatus)`
  font-size: 2.375rem;
`