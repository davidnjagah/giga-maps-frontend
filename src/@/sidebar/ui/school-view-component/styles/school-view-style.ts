import { Accordion } from "@carbon/react";
import styled from "styled-components";

import { Scroll } from "~/@/scroll";


export const MultischoolBottomInfoItem = styled.div`
margin-top:0.75rem;
margin-right:1.5rem;
display: flex;
    align-items: center;
  svg{
    width: 0.75rem;
height: 0.75rem;
    fill:${props => props.theme.text};
  }
`

export const MultischoolBottomInfo = styled.div`
      display: flex;
    padding-left: 1rem;
    flex-direction: column;
    .multi-school-bottom-info{
       display: flex;
    align-items: center;
    }
`

export const AccordionTitle = styled.span`
  position: relative;
  left: 2.5rem;
  
  display: block;
  top: -0.35rem;
  color:${props => props.theme.text};
  font-size: 0.875rem;
font-weight: 600;
line-height: 1.125rem; 
letter-spacing: 0.01rem;
`

export const ConnectivityStatusCircle = styled.div<{ $color?: string; }>`
  background: ${props => props.$color} ;
  width: 0.5rem;
  height: 0.5rem;
  border-radius: 1.25rem;
  opacity: 0.65;
`
export const SchoolTitleWrapper = styled.div`
display:flex;
align-items:center;
margin-left: 0.3rem;
p{
  width: 11rem;
  overflow: hidden;
  text-overflow: ellipsis;
  color:  ${props => props.theme.text};
font-family: "Open Sans";
font-size: 0.875rem;
font-style: normal;
font-weight: 600;
line-height: 1.125rem; /* 128.571% */
letter-spacing: 0.01rem;
margin-right: 0.35rem;
}

@media(min-width:1584px){
  p{
    width: 11rem;
  }
}
`

export const MultiSchoolAccodion = styled.div`
    width: 17.5rem;
    @media (min-width:1584px){
      width: 18rem;
    }

    .cds--accordion__heading{
      justify-content: space-between !important;

    }
.cds--accordion__heading:focus::before {
    box-sizing: border-box;
    border: none;
}
    @media (max-width:768px){
      width: 100%;
    }
`

export const MultiSchoolContainer = styled.div`
  width: 100%;
  margin: 0;
  position: relative;
`

export const SchoolMultiAccordion = styled(Accordion)`
    .cds--accordion__title {
      padding-bottom: 0.5rem;
      width: 23rem;
      white-space: nowrap;
      margin: 0;
      direction: ltr;
      padding-left: 0.125rem;
      padding-top: 1rem;
    }
  .cds--accordion__item--active,
  .cds--accordion__item {
    border-top: none;
    /* border-bottom: none; */
  }

  .cds--accordion__title {
    // margin: 0.88rem 0rem 0rem 1rem;
    padding-right: 0;
  }
  .cds--accordion__content {
    padding: 0;
    margin-left: 0;
  }

  .cds--accordion__arrow {
    align-self: flex-start;
    margin-top: 0.7rem;
    fill:${props => props.theme.text};
  }
  .cds--accordion__item:last-child{
    border-top: 1px solid ${props => props.theme.schoolListBack} ;
    border-bottom: 1px solid ${props => props.theme.schoolListBack} ;
    padding-top: 0.4rem;
  }

  .cds--accordion__heading:hover::before {
    background-color: ${props => props.theme.main};
  }
`

export const SchoolScrollView = styled(Scroll)`
    //max-height: calc(100% - 8rem);   
     //max-height: calc(100vh - 19rem);
    //max-height: calc(100vh - 16.5rem);
    max-height: 100% !important;
    height: calc(100% - 2rem) !important;
    
    @media (max-width:768px){
      // max-height: calc(100% - 26rem) !important;
      // max-height: 100% !important;
      height: calc(100% - 7rem) !important;
    }
`

export const SchoolDetailTitle = styled.h2`
    color:  ${props => props.theme.text};
    font-family: Open Sans;
    font-size: 0.875rem;
    font-style: normal;
    font-weight: 500;
    line-height: 1.25rem;
    letter-spacing: 0.01rem;
`
export const SchoolDetailInfo = styled.h2`
  display: grid;
  grid-template-columns: 1fr 1fr;
  margin: 0 auto;
  // padding: 0rem 0rem 0.88rem 0rem;
  width: inherit;
  // max-height: 11.2rem;
  overflow-y: auto;
  width: 100%;
`

export const SchoolDetailItem = styled.div`
  padding: 0.75rem 0.5rem;
  overflow: hidden;
  word-wrap: break-word;
  padding-left: 0;
p {
  color:   ${props => props.theme.schoolId};
  font-family: Open Sans;
  font-size: 0.75rem;
  font-style: normal;
  font-weight: 400;
  line-height: 1rem;
  margin-bottom: 0.25rem;
  /* 133.333% */
}

span {
  display: block;
  color: ${props => props.theme.text};
  font-family: Open Sans;
  font-size: 0.875rem;
  font-style: normal;
  font-weight: 400;
  line-height: 1.125rem;
  /* 128.571% */
}
`

export const SchoolInformationWrapper = styled.div`
  padding: 1rem;
  color:${props => props.theme.text};
  padding-top: 2rem;
`

export const SchoolInfoContainer = styled.div`
display: grid;
  grid-template-columns: 1fr 1fr;
    width: 100%;
`
export const SingleInfoContainer = styled.div<{ $width?: boolean }>`
    display: flex;
    align-items: center;
    margin-top: 0.75rem;
svg{
  width: 0.75rem;
height: 0.75rem;
fill:${props => props.theme.text};
margin-right:0.25rem;
}

p{
  color: ${props => props.theme.titleDesc};
font-size: 0.75rem;
font-style: normal;
font-weight: 400;
line-height: 1.125rem; 
overflow: hidden;
text-overflow: ellipsis;
text-transform: capitalize;
width: ${props => props.$width ? "100%" : "6.4rem"};
.lowercase {
  text-transform: lowercase;
}

}
`