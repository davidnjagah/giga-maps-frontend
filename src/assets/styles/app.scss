// @import 'init';
// @import 'main';

// .app {
//   position: relative;
//   display: flex;
//   flex-flow: column nowrap;
//   align-items: stretch;
//   width: 100%;
//   height: 100%;

//   &--inner {
//     .header {
//       background-color: $header-background-color;
//     }

//     .header__button {
//       @include media-breakpoint-down(md) {
//         display: none;
//       }
//     }
//   }

//   &--home {
//     .header {
//       @include media-breakpoint-up(md) {
//         height: 0;
//       }
//     }

//     .header__logo {
//       position: absolute;
//       top: 3.2rem;
//       transform: translateY(-50%);
//     }

//     .header__button {
//       position: absolute;
//       top: 3.2rem;
//       right: ($grid-gutter-width-lg * 0.5);
//       transform: translateY(-50%);
//     }

//     .download__button {
//       position: absolute;

//       transform: translateY(-50%);
//     }

//     .content {
//       height: 100%;
//     }
//   }
// }

// .mapboxgl-ctrl.mapboxgl-ctrl-attrib {
//   height: 1.3rem !important;
// }


// .school-mapped {
//   // margin-right: 8rem !important;
// }

// .footer-slider {
//   display: flex;
//   align-items: center;

//   // position: absolute;
//   // left: 23rem;
//   // bottom: 0rem;
//   .cds--form-item {
//     flex-direction: initial;
//   }

//   .cds--slider__range-label {
//     color: #7E7E7E !important;
//     font-size: .75rem !important;
//   }

//   .cds--slider__filled-track,
//   .cds--slider__thumb {
//     background: #7E7E7E !important;
//   }
// }

// .logo {
//   color: $text-color-light;
//   font-weight: 500;
//   font-size: 1.2rem;
//   letter-spacing: 0.126rem;
//   text-transform: uppercase;
//   text-decoration: none;
//   background-color: transparent;
//   border: none;
//   outline: none;
//   cursor: pointer;
//   //noinspection CssBrowserCompatibilityForProperties
//   user-select: none;

//   @include media-breakpoint-up(md) {
//     font-size: 1.7rem;
//     letter-spacing: 0.143rem;
//   }
// }

// .map-placeholder {
//   position: absolute;
//   top: 16.4rem;
//   right: 0;
//   bottom: 4.5rem;
//   left: 0;
//   background-color: #808080;

//   @include media-breakpoint-up(md) {
//     top: 0;
//     bottom: 0;
//   }
// }

// .mapboxgl-ctrl.mapboxgl-ctrl-attrib .mapboxgl-ctrl-attrib-inner {
//   margin-bottom: 5.5rem;
// }

// .mapboxgl-ctrl.mapboxgl-ctrl-attrib.mapboxgl-compact {
//   margin-bottom: 6rem;
//   display: none;
// }

// .mapboxgl-ctrl .mapboxgl-ctrl-logo {
//   margin-bottom: 1.5rem;
//   display: none;
// }

// .mapboxgl-canvas {
//   outline: none;
// }