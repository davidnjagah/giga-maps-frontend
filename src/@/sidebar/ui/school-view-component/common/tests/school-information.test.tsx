import { render } from '@testing-library/react';
import { testWrapper } from '~/tests/jest-wrapper';
import { SchoolInformation } from '../school-information.view';
import { SchoolDetailItem, SchoolDetailTitle } from '../../styles/school-view-style';

describe('School information', () => {
  it('should render component', () => {
    const { asFragment } = render(testWrapper(<SchoolInformation />));
    expect(asFragment).toMatchSnapshot();
  });

  it('should render component', () => {
    const { asFragment } = render(testWrapper(<SchoolDetailTitle />));
    expect(asFragment).toMatchSnapshot();
  });

  it('should render component', () => {
    const { asFragment } = render(testWrapper(<SchoolDetailItem />));
    expect(asFragment).toMatchSnapshot();
  });
});
