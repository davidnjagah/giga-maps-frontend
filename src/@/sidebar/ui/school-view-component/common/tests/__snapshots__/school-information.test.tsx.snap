// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`School information should render component 1`] = `[Function]`;

exports[`School information should render component 2`] = `[Function]`;

exports[`School information should render component 3`] = `[Function]`;
